daphne/src/roles_test.rs

```diff
@@ -499,6 +501,49 @@ async fn get_reports_empty_response() {
     assert_eq!(reports.len(), 0);
 }
 
+// Test a successful collect request submission.
+// This checks that the Leader reponds with the correct collect ID.
+// TODO(nakatsuka-y) Implement a test case when a Leader rejects a collect request.
```

Comment:
* There is no "correct ID", right? What we're checking here is that the collect URI uniquely identifies the collect request.
* nit: The first two lines should be in the same paragraph (i.e., remove newline between them).
* nit: I like to put a blank line between the comment and the TODO:

```rust
// A comment that describes the function.
//
// TODO(blah) Blah.
```

Reply:
You are correct. I've updated this comment and others based on your feedback.